sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -421,6 +434,31 @@ export interface FileReadToBufferOptions extends CommonOptions {
     onProgress?: (progress: TransferProgressEvent) => void;
 }
 
+// @public
+export interface FileSetExpiryHeaders {
+    clientRequestId?: string;
+    date?: Date;
+    // (undocumented)
+    errorCode?: string;
+    etag?: string;
+    lastModified?: Date;
+    requestId?: string;
+    version?: string;
+}
+
+// @public
+export interface FileSetExpiryOptions extends CommonOptions {
+    abortSignal?: AbortSignalLike;
+    expiresOn?: number | Date;
```

Comment:
I'd prefer if this was Date only :)

Reply:
When using relative mode - "RelativeToCreation" or "RelativeToNow", it's the value of milliseconds elapsed from the base time. Using a Date for this scenario seems be not quite straight-forward.

Reply:
Hmm that makes sense, but in that case shouldn't the name be something like `timeUntilExpiredInMs`?

Reply:
I think we should either make it always the Date of expiry or we should have a different property name in the relative cases

Reply:
How about this?
```ts
export interface FileSetExpiryOptions extends CommonOptions {
    abortSignal?: AbortSignalLike;
    expiresOn?: Date;
    timeToExpireInMs?: number;
```

Reply:
Perfect!

Reply:
No concerns from me with your proposed change @ljian3377.